streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimestampedTupleForwarderTest.java

```diff
@@ -66,11 +66,11 @@ private void shouldForwardRecordsIfWrappedStateStoreDoesNotCache(final boolean s
 
         expect(store.setFlushListener(null, sendOldValues)).andReturn(false);
         if (sendOldValues) {
-            context.forward("key1", new Change<>("newValue1",  "oldValue1"));
-            context.forward("key2", new Change<>("newValue2",  "oldValue2"), To.all().withTimestamp(42L));
+            context.forward(new Record<>("key1", new Change<>("newValue1",  "oldValue1"), 0L));
```

Comment:
Same question about these timestamps... For the ones that weren't present before, does it matter what we set?

Reply:
not really, forwarders prefer either key (session window) or value (kv+timestamp) ts before using records.